chart/templates/blobserve-configmap.yaml

```diff
@@ -32,6 +32,11 @@ data:
                     "replacements": [
                         { "search": "vscode-webview.net", "replacement": "{{ .Values.hostname }}", "path": "/ide/out/vs/workbench/workbench.web.api.js" },
```

Comment:
this line does not seem to be appied anymore

<img width="482" alt="Screenshot 2021-08-31 at 15 48 01" src="https://user-images.githubusercontent.com/3082655/131514186-3314dbdd-1e31-40be-ab76-85d182dd4ebb.png">

You can reproduce it with `Simple Browser: Show` command.

Reply:
@corneliusludmann I tried in another prev env and could not reproduce it:
<img width="1904" alt="Screenshot 2021-09-03 at 08 37 18" src="https://user-images.githubusercontent.com/3082655/131961352-8c965354-d2b4-440d-885f-79dd61124eaa.png">

I think it happens only in this PR and caused by some bug in the blobserve replacement logic.

Reply:
Indeed, the paths are stored as key in a map so that the second entry replaces the first one.

https://github.com/gitpod-io/gitpod/blob/5c73a864d94cc9436cdb9341b6f1803ea2cd7a81/components/blobserve/pkg/blobserve/refstore.go#L29-L31

Reply:
@akosyakov Fixed in 69a8c54e727da6f1fcf483d63e38bff77029527b.